modules/bmo-about-the-baremetalhost-resource.adoc

```diff
@@ -0,0 +1,270 @@
+// This is included in the following assemblies:
+//
+// post_installation_configuration/bare-metal-configuration.adoc
+
+:_content-type: REFERENCE
+[id="about-the-baremetalhost-resource_{context}"]
+= About the BareMetalHost resource
+
+Metal^3^ introduces the concept of the `BareMetalHost` resource, which defines a physical host and its properties. The `BareMetalHost` resource contains two sections:
+
+. The bare metal host specification.
+. The bare metal host's current status.
+
+[discrete]
+== The BareMetalHost specification
+
+The `spec` section of the `BareMetalHost` resource defines the desired state of the host.
+
+.BareMetalHost spec
+[options="header"]
+|====
+|Parameters |Description
+
+a|
+----
+bmc:
+  address:
+  credentialsName:
+  disableCertificateVerification:
+----
+a| The `bmc` configuration setting contains the connection information for the baseboard management controller (BMC) on the host. The fields are:
+
+* `address`: The URL for communicating with the BMC controller.
+
+* `credentialsName`: A reference to a secret containing the username and password for the BMC.
+
+* `disableCertificateVerification`: A boolean to skip certificate validation when set to `true`.
+
+| `online`
```

Comment:
Nit - this should be after hardwareProfile as it shows up alphabetically in the resource